test/com/google/javascript/jscomp/integration/ProductionCoverageInstrumentationPassIntegrationTest.java

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright 2009 The Closure Compiler Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.javascript.jscomp.integration;
+
+import com.google.javascript.jscomp.BlackHoleErrorManager;
+import com.google.javascript.jscomp.Compiler;
+import com.google.javascript.jscomp.CompilerOptions;
+import com.google.javascript.jscomp.CompilerOptions.InstrumentOption;
+import com.google.javascript.jscomp.CompilerOptions.LanguageMode;
+import com.google.javascript.jscomp.SourceFile;
+import com.google.javascript.jscomp.testing.NoninjectingCompiler;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class ProductionCoverageInstrumentationPassIntegrationTest extends
+    IntegrationTestCase {
+
+
+  private final String instrumentCodeSource =
+      lines(
+          "goog.module('instrument.code');",
+          "class InstrumentCode {",
+          "instrumentCode(a,b) {}",
```

Comment:
minor issue: I'd find this code + a few other places in this file more readable if you indented the source code. e.g.
```
          "goog.module('instrument.code');",
          "class InstrumentCode {",
          "  instrumentCode(a,b) {}",          
          "}",
```